docs/proposals-accepted/202304-cardinalitymanagment.md

```diff
@@ -0,0 +1,132 @@
+---
+type: proposal
+title: Series Cardinality API
+status: accepted
+owner: jatinagwal
+menu: proposals-accepted
+---
+
+###  Related links/tickets
```

Comment:
@jatinagwal Can you please try to format the whole doc a little bit? Like keeping one space between words, one blank line between paragraphs, etc